File: app/src/main/java/com/moictab/curriculumvitae/model/Experience.java
package com.moictab.curriculumvitae.model;


import com.google.gson.annotations.SerializedName;

/**
 * Created by moict on 15/10/2017.
 */

public class Experience {

    @SerializedName("title")
    public String title;

    @SerializedName("place")
    public String place;

    @SerializedName("website")
    public String website;

    @SerializedName("period")
    public String period;

    @SerializedName("description")
    public String description;

    @SerializedName("image")
    public String image;
}

File: app/src/main/java/com/moictab/curriculumvitae/model/Experience.kt
package com.moictab.curriculumvitae.model

import com.google.gson.annotations.SerializedName

class Experience {
  @SerializedName("title")
  var title: String? = null
  @SerializedName("place")
  var place: String? = null
  @SerializedName("website")
  var website: String? = null
  @SerializedName("period")
  var period: String? = null
  @SerializedName("description")
  var description: String? = null
  @SerializedName("image")
  var image: String? = null
}
